### ISSUE TITLE
Add support for custom model fields

### ISSUE THREAD

**Issue by tsvetomir** [telerik/kendo-angular#1907]
## I'm submitting a...

* Suggestion for improvement  

## Current behavior
The Scheduler supports [binding to models](https://www.telerik.com/kendo-angular-ui/components/scheduler/data-binding/#toc-data-binding) of type [SchedulerEvent](https://www.telerik.com/kendo-angular-ui/components/scheduler/api/SchedulerEvent/).

The developer has the sole responsibility of converting their models to events and back.

## Expected behavior
The Scheduler should be able to take a field map and perform the mapping automatically. For example:

```ts
import { SchedulerModelFields } from '@progress/kendo-angular-scheduler';

@Component({
  selector: 'my-app',
  template: `
    <kendo-scheduler [modelFields]="myModelFields" ...>
        <kendo-scheduler-day-view>
        </kendo-scheduler-day-view>
    </kendo-scheduler>
  `
)
export class MyAppComponent {
  myModelFields: SchedulerModelFields = {
    id: 'TaskID',
    start: 'Start',
    end: 'End',
    isAllDay: 'AllDay',
    title: 'Title',
    recurrenceRule: 'RRule',
    recurrenceId: 'RID',
    recurrenceExceptions: 'RExceptions'
  }
}
```

**Comment by tsvetomir** [comment]
The original model will be accessible as `SchedulerEvent.dataItem`to allow displaying and editing custom fields.

**Comment by Raisolution** [comment]
Implemented in `v0.7.0`
